.github/workflows/reusable_fetch_content.yml

```diff
@@ -23,17 +23,20 @@ jobs:
         shell: bash
 
     steps:
-      - name: Add Repos for for gcc-13 and clang-16
+      - name: Add Repos for for gcc-13 and clang-16,17
         run: |
           # gcc-13
           sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+          
+          source /etc/os-release
```

Comment:
It's the first time I noticed this. Whatis it good for? what is in os-release?

Reply:
The value for `${UBUNTU_CODENAME}` thats used below to select the correct repo